portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Megha's Portfolio</h1>
+      <p>Thank you for checking out my portfolio! My name is Megha and I'm a rising junior at UC Irvine majoring in 
+        Computer Science and Engineering. I like to be involved on campus, to make my family and friends laugh, 
+        and to put together music playlists for my friends! 
+        I hope by going through this website, you can get to know a little more about me :)</p>
+      <img src="images/meghaPortfolio.jpg" class="mainPic"/>
+      <!--<p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="greeting-container"></div>-->
```

Comment:
It's better to remove old code rather than comment it out. You don't want commented out old code going into master. If you want to reference the original code for later it's better to just check the history of the file :)

Reply:
Noted! Just committed the change